hlx_statics/blocks/columns/columns.js

```diff
@@ -53,10 +55,34 @@ export default async function decorate(block) {
   });
 
   block.querySelectorAll('a').forEach((a) => {
-    a.classList.add('spectrum-Link', 'spectrum-Link--quiet');
+    if (!a.classList.contains('button')) {
+      a.classList.add('spectrum-Link', 'spectrum-Link--quiet');
+    }
     checkExternalLink(a);
   });
 
+  block.querySelectorAll('.button').forEach((button) => {
+    button.classList.add('spectrum-Button', 'spectrum-Button--sizeM');
+    if (button.parentElement.tagName.toLowerCase() !== 'strong') {
+      button.classList.add('spectrum-Button--secondary', 'spectrum-Button--outline');
+    } else {
+      button.parentElement.replaceWith(button);
+      button.classList.add('spectrum-Button--cta');
+    }
+  });
+
+  /* Stop here when metadata is `style: center` */
+  if (container.classList.contains('center')) {
+    return;
+  }
+
+  block.querySelectorAll('.columns > div > div:first-child').forEach((column) => {
```

Comment:
probably this could be merged into one query by adding` .columns > div > div:nth-child(1), .columns > div > div:nth-child(2)` and do an if-else statement when it has the class first-column

Reply:
I think here, the expected behaviour is to add for the first and second column classes just for those columns and, in case there are more than 2 columns to not add any extra class to them, the if-else wouldn't simplify the statements from what I see.

Reply:
It would be something like 
```
block.querySelectorAll('.columns > div > div:nth-child(1), .columns > div > div:nth-child(2)').forEach((column) => {
    if (column.classList.contains('first-column')) {
      column.classList.add('first-column');
    } else {
      column.classList.add('second-column');
    }
  });
```
and avoid the double query for first and second column. But I agree, it doesn't seem to simplify a lot the statement